package.json

```diff
@@ -64,6 +65,7 @@
     "babel-eslint": "^10.1.0",
     "caniuse-lite": "^1.0.30001255",
     "chai": "^4.3.4",
+    "cypress": "^7.7.0",
```

Comment:
Same thing as with `@babel/core`, Cypress' version is already set as a byproduct of listing `@vue/cli-plugin-e2e-cypress` so we shouldn't force set it (as `@vue/cli-plugin-e2e-cypress` has been tested with a specific version, and not necessarily the latest)